Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/Controllers/TestController.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.BotCommands;
+using System;
+using System.Collections.Generic;
+
+namespace Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.Controllers
+{
+    public abstract class TestController : IController
```

Comment:
jaki sens ma ta klasa?

Reply:
Jest potrzebna do testowania. Kiedy zostanie wykryta komenda i dobrze sparsowana to następnie jest wywoływana odpowiednia metoda z tej klasy i dzięki temu możemy sprawdzić jakie wartości zostały przekazane metodom a to nam daje to że możemy sprawdzić czy wartości zostały sparsowane dobrze. Chyba o to w tych testach chodzi, aby sprawdzić działanie klasy RunnerOfIBotCommandMethods.